cli/src/commit_templater.rs

```diff
@@ -1090,19 +1144,21 @@ fn builtin_commit_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, Comm
     map
 }
 
-// TODO: return Vec<String>
-fn extract_working_copies(repo: &dyn Repo, commit: &Commit) -> String {
-    let wc_commit_ids = repo.view().wc_commit_ids();
-    if wc_commit_ids.len() <= 1 {
-        return "".to_string();
+fn extract_working_copies(repo: &dyn Repo, commit: &Commit) -> Vec<Rc<WorkspaceRef>> {
+    let mut working_copies = Vec::new();
```

Comment:
nit: I would move this closer to the for loop. You can also rewrite the loop as `.iter().filter().map().collect()`.